teams.json

```diff
@@ -17,20 +17,40 @@
       "name": "code-build-run",
       "apis": [
         "appengine",
+        "artifactregistry",
         "cloudbuild",
         "cloudfunctions",
         "cloudscheduler",
         "cloudtasks",
         "container",
-        "containeranalysis",
         "run",
         "logging",
-        "clouderrorreporting"
+        "clouderrorreporting",
+        "containeranalysis",
+        "containerregistry"
       ],
       "repos": [
         "googleapis/nodejs-logging-winston",
         "googleapis/nodejs-logging-bunyan",
+        "googleapis/nodejs-cloudbuild",
+        "googleapis/java-cloudbuild",
+        "googleapis/python-cloudbuild",
+        "googleapis/java-containeranalysis",
+        "googleapis/python-containeranalysis",
+        "googleapis/google-cloud-php-container-analysis",
         "GoogleCloudPlatform/cloud-code-samples",
+        "googleapis/nodejs-containeranalysis",
+        "GoogleCloudPlatform/php-docker",
```

Comment:
Oh my.  So if you want to track these via sloth, you would also need to add them to `repos.json`.  Before doing that - I would suggest sitting down, and doing a full scale triage session where you pre-apply all the right labels.  Otherwise, every issue in these repositories will instantly go out of SLO and it will be baaaaaad.  I am happy to sit with you to help get the feel of how a triage with the labels go.